mirgecom/steppers.py

```diff
@@ -59,6 +58,17 @@ def advance_state(rhs, timestepper, checkpoint, get_timestep,
         Time at which to start
     istep: int
         Step number from which to start
+    pre_step_callback
+        An optional user-defined function to be called before the timestepper
+        is called for that particular step. A non-zero return code from this
+        function indicates that this function should stop gracefully.
+        Examples of such functions include visualization, io, or restart
+        checkpoints.
+    post_step_callback
+        An optional user-defined function to be called after the timestepper
+        is called for that particular step. A non-zero return code from this
+        function indicates that this function should stop gracefully.
+        Examples of such functions include applying modal filtering or limiters.
```

Comment:
Document signature of the callbacks.

Reply:
Done: https://github.com/illinois-ceesd/mirgecom/pull/370/commits/b914d3dd258983b3bbf07a5e315a85579b5565bc